spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -871,6 +894,20 @@ public void setMissingQueuesFatal(boolean missingQueuesFatal) {
 
 	}
 
+	public static class StreamContainer extends BaseContainer {
+
+		boolean nativeListener;
```

Comment:
This property needs some javadoc that'll be captured in the configuration property metadata.